src/execution/riscv64/simulator-riscv64.cc

```diff
@@ -662,13 +617,12 @@ void RiscvDebugger::Debug() {
         PrintF("disasm [[<address/register>] <instructions>]\n");
         PrintF("  disassemble code, default is 10 instructions\n");
         PrintF("  from pc (alias 'di')\n");
+        PrintF("break / b\n");
+        PrintF("  break : list all breakpoints\n");
+        PrintF("  break <address> : set / enable / disable a breakpoint.\n");
```

Comment:
Suggest we support both `tbreak` (once a break point is hit, it will be disabled, see gdb tbreak) and `break` (regular breakpoint).